packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -424,4 +441,57 @@ export class GCFBootstrapper {
       });
     }
   }
+
+  /*
+   * Setting the process.env.WEBHOOK_TMP environment variable indicates
+   * that the webhook payload should be written to a tmp file in Cloud
+   * Storage. This allows us to circumvent the 100kb limit on Cloud Tasks.
+   *
+   * @param body
+   */
+  private async maybeWriteBodyToTmp(body: string): Promise<string> {
+    // Cloud tasks has a maximum payload size of 100kb, if a webhook payload is
+    // approaching this size, then serialize it to a temporary folder:
+    const MAX_PAYLOAD_SIZE = 65000;
+    if (process.env.WEBHOOK_TMP && body.length > MAX_PAYLOAD_SIZE) {
+      const tmp = `${Date.now()}-${v4()}.txt`;
```

Comment:
Honest question since you know more here - is `Date.now()` more safe than `uuid.v4()`?  Since there is no guarantee this is only process running at a time, I feel like having some kind of process based seed would be needed, no?

Reply:
I've been including a `uuid`, which is guaranteed not to collide, and a timestamp.

The reason for the timestamp is so that if we ever need to perform cleanup, we can infer when the object was written by name.

Reply:
Oh gosh I read right over that 🙈 Apologies!